src/app/shared/notification/notification.component.html

```diff
@@ -24,6 +24,16 @@ <h3>{{message.summary}}</h3>
   </button>
 </ng-template>
 
-
-
-
+<ng-template #problemToastContent let-message="message">
+  <h3>{{message.summary}}</h3>
+  <p id="message-details">{{message.detail}}</p>
+  <div id="message-errors" *ngFor="let error of message.data.errors">
+    <p>{{error}}</p>
+  </div>
+  <a pButton class="p-button-secondary"
```

Comment:
> Since it does not specify a link type, that page will be able to access the DOM of the origin page.
>
> https://github.com/iver-wharf/wharf-web/security/code-scanning/3?query=ref%3Arefs%2Fpull%2F54%2Fmerge+ref%3Arefs%2Fpull%2F54%2Fhead+ref%3Arefs%2Fheads%2Ffeature%2Fproblem-response-toast

Yo that sh*t's wild

We kind of know the URL will be safe, but it'll be nice to have that extra safe guard just in case

```suggestion
  <a pButton class="p-button-secondary"
     rel="noopener noreferrer"
```

Reply:
Huh, that's wild indeed.

Didn't see it because it's medium, and I'm assuming that's why the check was green.
Now I know to always expand the checks! 👍

Reply:
It was visible through the "File changes" tab, as if it was yet another review comment